cmd/res/configuration.yaml

```diff
@@ -0,0 +1,47 @@
+# REFERENCE: https://docs.edgexfoundry.org/3.0/microservices/device/Ch-DeviceServices/
+Writable:
+  LogLevel: INFO
+  Reading:
+    ReadingUnits: true
+  Telemetry:
+    Metrics:
+      EventsSent: false
+      ReadingsSent: false
+
+Service:
+  Host: localhost
+  Port: 59995
+  StartupMsg: device uart started
+  RequestTimeout: "5s"
+
+MessageBus:
+  Protocol: redis
+  Host: localhost # uncomment when running from command-line in hybrid mode
+  Port: 6379
+  Type: redis
+  BaseTopicPrefix: "edgex" # prepended to all topics as "edgex/<additional topic levels>  
+  Optional:
+    ClientId: device-uart
+
+Clients:
+  core-metadata:
+    Protocol: "http"
+    Host: "localhost"
+    Port: 59881
+
+Device:
+  DataTransform: true
+  MaxCmdOps: 128
+  MaxCmdValueLen: 256
+  ProfilesDir: "./res/profiles"
+  DevicesDir: "./res/devices"
```

Comment:
Note that we leave these so the can be overridden with Env Vars, even though they are in the common config.
See here for example: https://github.com/edgexfoundry/device-virtual-go/blob/main/cmd/res/configuration.yaml#L23-L26

Reply:
Retained.